gensim/test/test_utils.py

```diff
@@ -84,6 +85,48 @@ def test_decode_entities(self):
         expected = u'It\x92s the Year of the Horse. YES VIN DIESEL \U0001f64c \U0001f4af'
         self.assertEqual(utils.decode_htmlentities(body), expected)
 
+    def test_open_file_existent_file(self):
+        exc = False
+        try:
+            with utils.open_file(datapath('testcorpus.mm')):
+                pass
+        except Exception:
+            exc = True
+
+        self.assertFalse(exc)
+
+    def test_open_file_non_existent_file(self):
+        exc = False
+        try:
+            with utils.open_file(datapath('non_existent_file.txt')):
+                pass
```

Comment:
Please iterate through this file too (not enough only open it), here and everywhere.
Simplest way looks like like
```python
with utils.open_file("...") as infile:
    self.assertEqual(sum(1 for _ in infile), numer_of_lines_in_file)
```